com/metacube/EAD/Assignment 4/WebContent/SignUp.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>SignUp Page</title>
+        
+        <script  src="scripts/validate.js"></script>
+        
+        <link href="designing.css" rel="stylesheet">
+    </head>
+    
+    <body>
+        <table class="headTableStyle" border="1" style="width:100%">
+            <tr>
+                <td><img src="images/html5logo.png" style="width:50px"></td>
+                <td align = "right"><a href="Home.html">Home</a>
+                    <a href="#">About</a>
+                    <a href="SignUp.html">SignUp</a>
+                    <a href="Login.html">Login</a>
+                    <a href="#">ContactUs</a></td>
+            </tr>           
+        </table>
+        
+        <hr> <br><br><br><br>
+        
+        <form name="signUpForm" action="SignUpServlet" method="POST">
+            <table class="signUpFormStyle" border="0px" align="center" cellspacing="15px">
+            <caption><h1>Enter Details</h1></caption>
+                <tr>
+                    <td>First Name</td>             
+                    <td><input type="text" name="firstName" id="firstName" required onkeyup="validateFirstName()"></td>
+                </tr>
+                <tr>
+                    <td>LastName</td>               
+                    <td><input type="text" name="lastName" id="lastName" required onkeyup="validateLastName()"></td>
+                </tr>
+                <tr>
+                    <td>Date of Birth</td>              
+                    <td><input type="date" name="dob" id="dob" required onblur="validateDOB()"></td>
+                </tr>
+                <tr>
+                    <td>Contact Number</td>             
+                    <td><input type="number" name="number" id="number" required onkeyup="validateNumber()"></td>
+                </tr>
+                <tr>
+                    <td>Email</td>              
+                    <td><input type="text" name="email" id="email" required onkeyup="validateEmail()"></td>
+                </tr>
+                <tr>
+                    <td>Password</td>               
+                    <td><input type="password" name="password" id="password" required onkeyup="validatePassword()"></td>
```

Comment:
Add the error message next to the field in case the validation fails. In case of password particularly, add info message next to the field specifying the password matching criteria i.e. number of characters, caps or small letters, special characters,etc so that user is well informed what type of password they can create.